src/fsharp/service/FSharpCheckerResults.fsi

```diff
@@ -82,6 +82,15 @@ type public FSharpProjectOptions =
     /// Compute the project directory.
     member internal ProjectDirectory: string
 
+and [<NoComparison>] public FSharpReferencedProject =
+    internal
```

Comment:
Could you please make it public instead?

Reply:
I don't want the internal structure to be public. But, we should be able to create different kinds of `FSharpReferencedProject` instances.